tools/nbs_tools/decode_image/decode_bayer.py

```diff
@@ -0,0 +1,52 @@
+#!/usr/bin/env python3
+
+import numpy as np
+import tensorflow as tf
+
+from .fourcc import fourcc
+
+lookups = {}
+
+
+def decode_bayer(data, format, factor=2):
+    raw_img = tf.io.decode_jpeg(data, channels=1)
```

Comment:
This function should only decode bayer formats. Jpeg decompression should happen external to this

Reply:
It's actually not even decoding bayer, It's just repermuting the mosaic. The actual debayering is happening in the ffmpeg call